doc/source/conf.py

```diff
@@ -193,7 +193,8 @@ def extract_example_links(
     list
         List of example links.
     """
-    g = Github()
+    token = os.environ.get("GITHUB_TOKEN")
+    g = Github(token)
```

Comment:
Is this really required? I thought we were only downloading stuff from public sources

Reply:
@jorgepiloto I just tried it out for fun with this repo as `repo_fullname` and was able to retrieve `type`, `name` and `download_url`. The token isn't required on a public repo.

Reply:
Usually, we don't need it, but I often add this locally, especially when working with styles. Because I build multiple times, it becomes necessary after a certain amount.

Reply:
reverting this.

Reply:
> Usually, we don't need it, but I often add this locally, especially when working with styles. Because I build multiple times, it becomes necessary after a certain amount.

Oh! That's because you are hitting the max. number of allowed requests. Makes sense.

Thanks for removing it.